CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+## [1.2.34](https://github.com/ably/ably-java/tree/v1.2.34)
+
+[Full Changelog](https://github.com/ably/ably-java/compare/v1.2.33...v1.2.34)
+
+**Fixed bugs:**
+
+- Should send `DETACH` after receiving `ATTACHED` while in the `DETACHING` or `DETACHED` state \(`RTL5k`\) [\#846](https://github.com/ably/ably-java/issues/846)
+
+**Closed issues:**
+
+- LocalDevice reset will cause ClassCastException [\#985](https://github.com/ably/ably-java/issues/985)
```

Comment:
In ably-cocoa we remove all the noise here and only leave issues that make sense for the end-user if they choose to read this change log.

Reply:
Done

Reply:
Somehow GH doesn't show me `Resolve` button. Try to resolve it yourself @sacOO7